server/api/api.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const Sighting = require('../models/Sighting.js');
+const sightingsController = require('../lib/sightingsController.js');
+
+const router = express.Router();
+
+// TODO: Adjust API to return data or err instead of unnamed JSON
+async function getSightings(req, res) {
+	try {
+		const result = await sightingsController.getSightings();
+		console.log('got sightings!');
+		res.status(200).json(result);
+	} catch (err) {
+		console.log(err);
+		res.status(500).json(err);
+	}
+}
+
+router.get('/sighting', getSightings);
+
+async function addSighting(req, res) {
+	const newSight = new Sighting(req.body);
+
+	try {
+		const result = await sightingsController.addSighting(newSight);
+		res.status(201).json({ message: 'Save Success', data: result });
+	} catch (err) {
+		res.status(500).json(err);
+	}
+}
+
+router.post('/sighting', addSighting);
+
+async function deleteSighting(req, res) {
+	const { id } = req.params;
+	console.log(id);
+
+	try {
+		const result = await sightingsController.deleteSighting(id);
+		res.status(204).json({ message: 'Sighting successfully deleted', data: result });
+	} catch (err) {
+		res.status(500).json(err);
+	}
+}
+
+router.put('/sighting/:id', deleteSighting);
```

Comment:
uhhhhh, why is delete a `put`

Reply:
delete has it's own branch that I'll create a PR for once this one is done, but it's also a put in there.

Would you use a delete?

Reply:
100% yes

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/DELETE

Reply:
`router.put('/sighting/:id'` should be for modifying an existing sighting, like adding notes or something

Reply:
Got it, will change. The thought behind put is that we were "modifying" an existing entry. With delete specific, makes more sense to use that.

Reply:
Done